samples/SocialSample/Startup.cs

```diff
@@ -136,7 +136,7 @@ k web
                         JObject user = JObject.Parse(text);
 
                         var identity = new ClaimsIdentity(
-                            context.Options.AuthenticationType,
+                            context.Options.AuthenticationScheme,
```

Comment:
Do we have the need to differentiate which parts of the ClaimsPrincipal get persisted?

We can't add arbitrary claims to an identity, this has the potential to change (amongst other things): ClaimsPrincipal.IsInRole("foo"). Think - can I add an arbitrary sid to an NTToken?